### ISSUE TITLE
split层不支持

### ISSUE THREAD

**Issue by xingmurong** [xxradon/PytorchToCaffe#117]
x_front, _  = torch.split(x1, [5, 1], dim=1)转caffe不支持类似于切片操作
如果用x_front = torch.index_select(x1, 1, torch.tensor([0, 1, 2, 3, 4]))，后一层则提醒TypeError: None has type NoneType, but expected one of: bytes, unicode
请教下怎么解决
